docs/markdown/_include_qt_base.md

```diff
@@ -34,6 +34,10 @@ It takes no positional arguments, and the following keyword arguments:
 Compiles Qt's moc files (.moc) into header and/or source files. At least one of
 the keyword arguments `headers` and `sources` must be provided.
 
+*NOTE:* the `preprocess` method allows strings to be passed to
+`include_directories` for historical reasons, this method does not, and
+requires `IncludeDirectory` objects.
```

Comment:
Shouldn't this also support strings to be consistent with [`executable()` et al.](https://mesonbuild.com/Reference-manual.html#executable)? Asking because passing strings was a feature that was explicitly added there.

Reply:
Ah, I thought that had been nixed hard :(